test/SparqlAntlrParserTestHelpers.h

```diff
@@ -252,8 +252,8 @@ MATCHER_P(GroupByVariablesMatch, vars, "") {
 }
 
 MATCHER_P2(IsValues, vars, values, "") {
-  return (arg.value()._inlineValues._variables == vars) &&
-         (arg.value()._inlineValues._values == values);
+  return (arg._inlineValues._variables == vars) &&
```

Comment:
Add a TODO to refactor the Values/SparqlValues classes s.t. this matcher becomes trivial.